tools/simple-library-server/package-lock.json

```diff
@@ -1,956 +1,8 @@
 {
   "name": "simple-library-server",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```

Comment:
I'm not sure why the `packages` section was removed. I thought the lockfile requires it.

Reply:
Probably a merging mistake. Reverted in [199d19d](https://github.com/enso-org/enso/pull/3667/commits/199d19dfd028f2df5561ac6ac3d3b846d51e3e67)